src/utils/output.ts

```diff
@@ -86,189 +93,347 @@ export function toArray<T>(entryOrArray: T | T[] | undefined): T[] {
+
+    if (!deletions.length) return;
+
+    const columns = {
+      fullName: { header: 'Name' },
+      type: { header: 'Type' },
+      filePath: { header: 'Path' },
+    };
+
+    const options = { title: info('Deleted Source') };
+    CliUx.ux.log();
 
-  if (verbose) {
-    displayVerboseTestSuccesses(result);
-    displayVerboseTestCoverage(result);
+    table(deletions, columns, options);
   }
 
-  CliUx.ux.log();
-  CliUx.ux.log(info('Test Results Summary'));
-  const passing = result.response.numberTestsCompleted ?? 0;
-  const failing = result.response.numberTestErrors ?? 0;
-  const total = result.response.numberTestsTotal ?? 0;
-  const time = result.response.details.runTestResult.totalTime ?? 0;
-  CliUx.ux.log(`Passing: ${passing}`);
-  CliUx.ux.log(`Failing: ${failing}`);
-  CliUx.ux.log(`Total: ${total}`);
-  if (time) CliUx.ux.log(`Time: ${time}`);
-}
+  private displayTestResults(): void {
+    if (this.testLevel === TestLevel.NoTestRun) {
+      CliUx.ux.log();
+      return;
+    }
+
+    this.displayVerboseTestFailures();
+
+    if (this.verbosity === 'verbose') {
+      this.displayVerboseTestSuccesses();
+      this.displayVerboseTestCoverage();
+    }
 
-function displayVerboseTestFailures(result: DeployResult): void {
-  if (!result.response.numberTestErrors) return;
-  const failures = toArray(result.response.details?.runTestResult?.failures);
-  const failureCount = result.response.details.runTestResult?.numFailures;
-  const testFailures = sortTestResults(failures) as Failures[];
-  CliUx.ux.log();
-  CliUx.ux.log(error(`Test Failures [${failureCount}]`));
-  for (const test of testFailures) {
-    const testName = underline(`${test.name}.${test.methodName}`);
-    const stackTrace = test.stackTrace.replace(/\n/g, `${os.EOL}    `);
-    CliUx.ux.log(`• ${testName}`);
-    CliUx.ux.log(`  ${dim('message')}: ${test.message}`);
-    CliUx.ux.log(`  ${dim('stacktrace')}: ${os.EOL}    ${stackTrace}`);
     CliUx.ux.log();
+    CliUx.ux.log(info('Test Results Summary'));
+    const passing = this.result.response.numberTestsCompleted ?? 0;
+    const failing = this.result.response.numberTestErrors ?? 0;
+    const total = this.result.response.numberTestsTotal ?? 0;
+    const time = this.result.response.details.runTestResult.totalTime ?? 0;
+    CliUx.ux.log(`Passing: ${passing}`);
+    CliUx.ux.log(`Failing: ${failing}`);
+    CliUx.ux.log(`Total: ${total}`);
+    if (time) CliUx.ux.log(`Time: ${time}`);
+  }
+
+  private displayVerboseTestSuccesses(): void {
+    const successes = toArray(this.result.response.details.runTestResult?.successes);
+    if (successes.length > 0) {
+      const testSuccesses = sortTestResults(successes) as Successes[];
+      CliUx.ux.log();
+      CliUx.ux.log(success(`Test Success [${successes.length}]`));
+      for (const test of testSuccesses) {
+        const testName = underline(`${test.name}.${test.methodName}`);
+        CliUx.ux.log(`${check} ${testName}`);
```

Comment:
I know check is colored using color success, what about the name of the test?

Reply:
It's also the success color